### ISSUE TITLE
Windows build release date

### ISSUE THREAD

**Issue by RosadinTV** [streamlink/streamlink#78]
Hi, first of all, glad to see the Livestreamer project live again with your work.
My question is if you have an estimated release date for the Windows build ?
Thanks

**Comment by gravyboat** [comment]
We just merged https://github.com/streamlink/streamlink/pull/27 yesterday. Once @cdrage does a build the Windows installer should be available.

**Comment by cdrage** [comment]
Hey @gravyboat just saw it was merged. Anything else that needs to be done before a pypi/pip release + tagged release on GitHub?

Otherwise, I'll create a changelog and release another version of streamlink!

**Comment by gravyboat** [comment]
@cdrage I don't think so. Do we have documentation up for where to get/how to use the windows installer? @sbstp is there anything else we need to do to provide this? I noticed the Windows installer link on the docs is broken currently since it's still using the livestreamer docs.

**Comment by sbstp** [comment]
I guess fix the links and explain the difference between the Python 2 and the Python 3 version.

**Comment by RosadinTV** [comment]
Great, awesome support 😄

**Comment by gravyboat** [comment]
@sbstp Do the differences between the Python 2 and 3 versions really matter that much for Windows users since it's self contained? We'll need to do the build so the installer is available before we update the docs I think. I would rather have the executable available sooner so we can start testing than worry about documenting those few items unless you believe they are absolutely critical.

**Comment by sbstp** [comment]
Yeah, my issue with it is that it might be confusing to people when they choose which one to download. They might think, that the version must match their own Python installation, but it's just the embedded python's version. 

I'd honestly like to use the python 3 version as it's smaller, simpler and has better isolation.

**Comment by gravyboat** [comment]
@sbstp Why don't we just avoid providing the Python 2 version then when it comes to Windows then? I see no reason to support an older release of Python when we have this clean slate to work with. If there is no advantage to doing so, there's no reason to support it or provide it as an available download.

**Comment by Xeala** [comment]
Would be nice if you build a portable version for Windows too :)

**Comment by gravyboat** [comment]
@sbstp @cdrage Let's do the next release today if possible, with or without docs. We can document later but we need to get a Windows release out there for people to use.

**Comment by cdrage** [comment]
@gravyboat np, I'll do the release now then :)

**Comment by cdrage** [comment]
@gravyboat done!

I think this can be closed and a new ticket opened for updating Windows documentation :)

You can checkout the new release on the release page and install the updated streamlink via `pip install streamlink --upgrade`

**Comment by gravyboat** [comment]
Thanks @cdrage.

**Comment by thinkpad4** [comment]
There will be a windows EXE installer version soon, right?

**Comment by cdrage** [comment]
@thinkpad4 Ideally, just a single binary (.exe)

**Comment by RosadinTV** [comment]
@cdrage  I think that is better to have the possibility to edit Plugins and other stuff too, i mean a binary (.exe) with all the folder libraries accessible to the user (like the original livestreamer Project) instead of a single executable with all packed and inaccessible to edit.
Check this pic:
https://s9.postimg.org/9yog5h5vj/livestreamer_portable_with_folders.png
